developers/ide/eclipse.md

```diff
@@ -5,65 +5,153 @@ title: Eclipse IDE
 
-   ![startup tasks finished](./images/ide_setup_eclipse_tasks_finished.png)
+```
+|- org.openhab.demo.app
+|--- runtime
+|------- conf             Here you configure the manual text files
+|------- userdata         Here is the openHAB user data
+|------- logback.xml      XML file for logging options
+|--- app.bndrun           The file to start a debug session
+|--- pom.xml              The pom file with all dependencies for the demo project
+```
 
-1. After all tasks are finished, look for `app.bndrun` file in the Eclipse Package Explorer in `Infrastructure > launch > app` project folder.
-   Double click to open `app.bndrun` file (takes a few seconds)
+1. Import the addon in Eclipse.
+Either it's an existing addon or a new binding created with the skeleton script.
+Import the addon project via `File > Import... > General > Existing Projects into Workspace`.
+Importing an addon is necessary if you want to modify or debug the addon.
+It's also possible to run existing addons part of the SNAPSHOT release in the demo project without having it imported in Eclipse.
+Simply follow the next step to add the addon.
 
-1. Under Browse Repos search for the binding you want to run/debug (`astro` in our case) and add it to the "Run Requirements" list using drag&drop from the "Browse Repos" list:
-    ![Bndtools](images/ide_setup_eclipse_bndtools.png)   
+1. To let the demo project know about the addon, the addon must be added to the demo project `pom.xml`.
+Here is an example for the `astro` binding:
 
-    NOTE: If the binding you want to run/debug cannot be found in the Browse Repos list, or the list is empty, then follow these steps to rebuild the list of bindings in the BOM (Bill Of Material) and make your new binding visible from the IDE launch configuration:
+```xml
+    <dependency>
+      <groupId>org.openhab.addons.bundles</groupId>
+      <artifactId>org.openhab.binding.astro</artifactId>
+      <version>${project.version}</version>
+      <scope>runtime</scope>
+    </dependency>
+```
 
-    - using Git bash console go to `git\openhab2-addons\bom\openhab-addons`
-    - recreate the BOM with: `mvn -DskipChecks -DskipTests clean install`
-    - go back to Eclipse, close and re-open the `app.bndrun` file and now you should be able to find the binding in the list
+3. To debug the addon with the `app.bdnrun` run configuration. 
+Double click to open `app.bndrun` file (takes a few seconds):
+
+    ![Bndtools](images/ide_debug_eclipse_1_bndtools.png)
+
+1. Under `Browse Repos` search for the binding you want to run/debug (`astro` in our case) and add it to the `Run Requirements` list using drag&drop from the `Browse Repos` list:
+
+    **NOTE:** If the binding you want to run/debug cannot be found in the Browse Repos list, or the list is  empty, then it's likely either the `pom.xml` of the demo project contains an error or there is a problem with your project.
+    You can check your project by
```

Comment:
Does this refer to the image underneath? I don’t understand this. It looks like an unfinished sentence.

Reply:
From the preview i would say its related to the following image, but it still seems a bit unclear while reading through.

Maybe somethign like `You can check you project, by starting a resolve.` sounds better.